imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -112,14 +115,15 @@ class ProductDetailContainer extends Component {
             [oppositeDirection]: "auto",
             [direction]: 0
           }, 600)
-          .delay(4000)
+          .delay(2000)
           .animate({
             [oppositeDirection]: "auto",
             [direction]: -alertWidth
           }, {
             duration: 600,
             complete() {
```

Comment:
Enabling the button at the start of the animation will defeat this solution as it will re-create the problem since users will be able to click `Add To Cart` rapidly.